service/Microsoft.Azure.Devices/AmqpServiceClient.cs

```diff
@@ -101,8 +101,10 @@ public SendingAmqpLink SendingLink
 
         public async override Task CloseAsync()
```

Comment:
Yes. Since Dispose() is guaranteed to run at the end, no need to invoke it from CloseAsync()

---
In reply to: [104151627](https://github.com/Azure/azure-iot-sdk-csharp/pull/57#discussion_r104151627) [](ancestors = 104151627)